server/service/src/display_settings_service.rs

```diff
@@ -0,0 +1,68 @@
+use repository::{KeyValueStoreRepository, KeyValueType, RepositoryError};
+
+use crate::{service_provider::ServiceContext, settings::DisplaySettings};
+
+pub struct UpdateResult {
+    pub logo: bool,
+    pub theme: bool,
+}
+
+pub trait DisplaySettingsServiceTrait: Sync + Send {
+    /// Loads display settings from the DB
+    fn display_settings(
+        &self,
+        ctx: &ServiceContext,
+    ) -> Result<Option<DisplaySettings>, RepositoryError> {
+        let key_value_store = KeyValueStoreRepository::new(&ctx.connection);
+
+        let custom_logo = key_value_store.get_string(KeyValueType::SettingsDisplayCustomLogo)?;
+        let custom_theme = key_value_store.get_string(KeyValueType::SettingsDisplayCustomTheme)?;
+        let make_settings = || {
```

Comment:
Don't think we need a closure here, also I think it's ok to return DisplaySettings non optional since custom_log and custom_theme would be optional.

Reply:
ah yes. thanks